db-scripts/src/main/resources/migration.sql

```diff
@@ -944,8 +944,9 @@ ALTER TABLE `cancer_study` DROP COLUMN `SHORT_NAME`;
 -- 2.12.9 for all changes
 
 -- WARNING: this will drop column TYPE_OF_CANCER_ID from table sample
-ALTER TABLE `sample`
-DROP FOREIGN KEY `sample_ibfk_2` ;
+-- the foreign key can exist as sample_ibfk_1 or sample_ibfk_2
+-- so removing the fk in check_and_remove_foreign_keys_sample_ibfk
+-- once the fk is removed, column can be dropped
```

Comment:
```suggestion
-- the foreign key on TYPE_OF_CANCER_ID in the sample table is removed with the migrate_db.py script. This is because the name can be either sample_ibfk_1  or sample_ibfk_2
```